custom_components/tuya_local/devices/petsnowy_snowplus_catlitter.yaml

```diff
@@ -0,0 +1,214 @@
+        name: button
+  - entity: switch
+    name: Do not disturb
+    icon: "mdi:power-sleep"
+    category: config
+    dps:
+      - id: 10
+        type: boolean
+        name: switch
+      - id: 106
+        type: string
+        optional: true
+        name: sensor
+  - entity: switch
+    name: Light
+    icon: "mdi:lightbulb"
+    category: config
+    dps:
+      - id: 16
+        type: boolean
+        name: switch
+  - entity: button
+    name: Deodorization
+    icon: "mdi:tailwind"
+    dps:
+      - id: 17
+        type: boolean
+        name: button
+  - entity: sensor
+    name: Notifications
+    class: enum
+    category: diagnostic
+    icon: "mdi:eye-circle-outline"
+    dps:
+      - id: 21
+        name: sensor
+        type: bitfield
+        mapping:
+          - dps_val: 1
+            value: Manual cleaning completed
+          - dps_val: 2
+            value: Automatic cleaning completed
+          - dps_val: 4
+            value: Timed cleaning completed
+          - dps_val: 8
+            value: Manual cleaning canceled
+          - dps_val: 16
+            value: Manual deodorization completed
+          - dps_val: 32
+            value: Automatic deodorization completed
+          - dps_val: 64
+            value: Timed deodorization completed
+          - dps_val: 128
+            value: Emptying cat litter completed
+          - dps_val: 256
+            value: Cat litter weight reset
+          - dps_val: 512
+            value: Self-check normal
+          - dps_val: 1024
+            value: Button locked
+          - dps_val: 2048
+            value: Button unlocked
+          - dps_val: 4096
+            value: Indicator light on
+          - dps_val: 8192
+            value: Indicator light off
+          - dps_val: 16384
+            value: Pause cleaning
+          - dps_val: 32768
+            value: Sleep mode on
+          - dps_val: 65536
+            value: Sleep mode off
+          - dps_val: 131072
+            value: Cancel emptying cat litter
+          - dps_val: 262144
+            value: Change cat litter
+          - value: Unknown
+  - entity: binary_sensor
+    class: problem
+    category: diagnostic
+    dps:
+      - id: 22
+        name: sensor
+        type: bitfield
+        mapping:
+          - dps_val: 0
+            value: false
+          - value: true
+      - id: 22
+        name: description
+        type: bitfield
+        mapping:
+          - dps_val: 0
+            value: OK
+          - dps_val: 1
+            value: Top cover fault
+          - dps_val: 2
+            value: Drawer fault
+          - dps_val: 4
+            value: Drawer full
+          - dps_val: 8
+            value: Stuck cat
+          - dps_val: 16
+            value: Checking fault
+          - dps_val: 32
+            value: Stay long
+          - dps_val: 64
+            value: Trouble removal
+          - value: Unknown
+  - entity: sensor
+    name: Status
+    class: enum

```

Comment:
enum requires mapping to know what values are output. Either remove the class, or add the mapping (without the class, I think HA will not try to keep a history)